source/libraries/HelperFunctions.ts

```diff
@@ -22,16 +22,21 @@ interface RpcClientOptions {
 }
 
 /**
- * Pads a string with zeros on the left and converts it to hexidecimal format.
+ * Pads a string with zeros (on the left by default) and converts it to hexidecimal format.
  * @param stringToEncode ASCII string to be padded & hexlified
  * @param length Desired length of the returned string (minus the "0x")
  */
-export async function padAndHexlify(stringToEncode: string, length: number): Promise<string> {
+export async function padAndHexlify(stringToEncode: string, length: number, direction: string = "left"): Promise<string> {
```

Comment:
THis doesn't `await` so it doesn't need to be `async`.  Making it a normal function will allow you to change the return type to just `string` and make it so you don't have to `await` the method when you call it, allowing it to be inlined easier.

Reply:
Ah, yeah.  Not sure why I originally made this async.  I'll change it.